.github/workflows/mattermost.yml

```diff
@@ -0,0 +1,41 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Mattermost_message
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the develop branch
+  push:
+    branches: [ feature-CI_mattermost_messages, develop, main ]
+  pull_request:
+    branches: [ feature-CI_mattermost_messages, develop, main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: push  
+        uses: Mitigram/gh-action-mattermost@main
+        if: github.event_name == 'push'
+        with: 
+          username: t8y
+          url: ${{ secrets.MATTERMOST_WEBHOOK_URL }}
+          text: ${{ github.event.pusher.name }} pushed into ${{ github.event.ref }}. See ${{ github.event.compare }} for more details.
```

Comment:
imho it is too much to trigger this for each push.
For a typical code review process this would spam our Chat.

Reply:
We can restrict it to certain actions. All actions possible are listed here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
I think, that opened, reopend, assigned and edited are probably the most important ones. We can restrict it to these cases.

Reply:
ah, wait.
This will later only fire when pushed into develop or main.
Not if we have a PR into develop and smth is pushed to the PR.

I guess then its fine.